Dockerfile

```
@@ -7,6 +7,7 @@ RUN ln -s /lib/libc.musl-x86_64.so.1 /lib/ld-linux-x86-64.so.2
 
 WORKDIR /ultralight
 
+COPY .git .git
```

```suggestion
COPY .git /.git
```
This should copy the .git folder into the root directory, because ultralight is being ran from the / directory so node will look for `.git` in the / directory